iot/api-client/end_to_end_example/cloudiot_pubsub_example_server.py

```diff
@@ -56,10 +56,9 @@
 
 class Server(object):
     """Represents the state of the server."""
-
-    def __init__(self, service_account_json):
-        credentials = ServiceAccountCredentials.from_json_keyfile_name(
-            service_account_json, API_SCOPES)
+def __init__(self, service_account_json):        
+        from google.auth import compute_engine
```

Comment:
This looks like it's the content from the codelab that I authored for Google I/O this year -- this sample (as opposed to the codelab) is more flexible without this import and credential initialization.